toolkit/tarball.py

```diff
@@ -106,7 +106,7 @@ def getnames(self):
 
     def read(self, arcname):
         """Returns sring with content of given file from tarball."""
```

Comment:
This wasn't your fault though but could you change the `sring` in the docstring to `string`? Thanks.

Reply:
Done in 8853fb7